pkg/R/SQLContext.R

```diff
@@ -36,12 +36,10 @@ jsonFile <- function(sqlCtx, path) {
 #' @export
```

Comment:
@evertlammerts Could you also update the doc above this to indicate `...` is for paths to parquet files ?